### PR TITLE
modifying the comparison document

### PR DESCRIPTION
**Pull request by shubhamsharan21**
*Issue #, if available:* NA

*Description of changes:* As there is a section for Observability using CloudWatch in EKS Anywhere, correcting the comparison doc where only 3rd party solution is written for monitoring and logging.

*Testing (if applicable):* NA

*Documentation added/planned (if applicable):* Modified

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by chrisnegus — COMMENTED** [review]
I'm suggesting removing the link to the new CloudWatch docs. We don't have links to documentation for other features in EKS and EKS-A, so that sort of sticks out. I'll create an issue to add links to all features named in the table, so team members can consider it.

**Comment by chrisnegus** [comment]
@csplinter I want to double check that you agree that CloudWatch is now an officially supported logging and monitoring feature for EKS-A, so I asked for you to review.

File: docs/content/en/docs/concepts/eksafeatures.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ To learn more about Amazon EKS, see [Amazon Elastic Kubernetes Service](https://
 | Command line interface (CLI)  | `eksctl` (OSS command line tool)        | `eksctl` (OSS command line tool) |
 | Console view for Kubernetes objects | Optional EKS console connection using EKS Connector (public preview) | Native EKS console connection|
 | Infrastructure-as-code        | Cluster manifest, Kubernetes controllers, [3rd-party solutions](https://aws.amazon.com/eks/eks-anywhere/partners/)            | AWS CloudFormation, [3rd-party solutions](https://aws.amazon.com/eks/partners/) |
-| Logging and monitoring        | [3rd-party solutions](https://aws.amazon.com/eks/eks-anywhere/partners/)            | CloudWatch, CloudTrail, [3rd-party solutions](https://aws.amazon.com/eks/partners/) |
+| Logging and monitoring        | [CloudWatch]({{< relref "../clustermgmt/observability/" >}}), [3rd-party solutions](https://aws.amazon.com/eks/eks-anywhere/partners/)            | CloudWatch, CloudTrail, [3rd-party solutions](https://aws.amazon.com/eks/partners/) |
```
[the remark this answers is not in the document]
Comment (shubhamsharan21) [reply docs/content/en/docs/concepts/eksafeatures.md:41]: Understood, thanks Chris. Removing the URL and keeping it plain

**Review by csplinter — APPROVED** [review]

**Pull request merged**